tap_frontapp/http.py

```diff
@@ -56,14 +54,20 @@ def request(self, method, path, **kwargs):
                 #so here we just run the request again
                 time.sleep(2)
                 response = requests.request(method, self.url(path), **kwargs)
-
+        elif path == '/conversations' or '/contacts/' in path:
```

Comment:
It looks like the rate limiting should already be handled based on [this code](https://github.com/singer-io/tap-frontapp/blob/9f39cb03d70dbc36fc126b7dd3dc05970a39d9a1/tap_frontapp/http.py#L36-L39). Could you just ensure that conversations and contacts requests are not caught by the `if` and `else` clauses here?

It would also be helpful if you could supply (redacted) logs showing these requests completing successfully over a minute without a 429 error.